### PR TITLE
use addon for travis apt package instead of sudo

### LINKED ISSUE #232: CSlang release fails
0.00s$ sudo apt-get install python-pip && ant -f build/ant.xml
      
sudo: must be setuid root
The command "sudo apt-get install python-pip && ant -f build/ant.xml
      " failed and exited with 1 during .
Your build has been stopped.

https://travis-ci.org/CloudSlang/cloud-slang/builds/57390557

Issue seems to be in the flow...

### PR DESCRIPTION
**Pull request by orius123**
hope it fixes #232

### CODE REVIEW

**Comment by orius123** [comment]
@meirwah need to merge and release in order to verify it

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,8 @@
 language: java
 
-after_success:
-  - mvn clean cobertura:cobertura coveralls:report
-
 before_deploy: >
-  sudo apt-get install python-pip &&
```
Comment (meirwah) [inline .travis.yml]: so where does this happen?

Comment (orius123) [inline .travis.yml]: according to travis-ci/travis-ci#3575 it is pre installed on every image

**Comment by meirwah** [comment]
:+1:

**Pull request merged**